config/src/config/execution_config.rs

```diff
@@ -19,6 +19,7 @@ const GENESIS_DEFAULT: &str = "genesis.blob";
 pub struct ExecutionConfig {
     #[serde(skip)]
     pub genesis: Option<Transaction>,
+    pub sign_vote: bool,
```

Comment:
naming could be improved... sign execution results etc

Reply:
but what I signed inside is `vote_proposal.hash()`